src/tools/genpolicy/src/utils.rs

```diff
@@ -97,6 +106,10 @@ impl Config {
     pub fn new() -> Self {
         let args = CommandLineOptions::parse();
 
+        if args.config_map_file.is_some() {
```

Comment:
Let's keep the old -c and call all of these 3 types of files "configuration files" instead of "resource files".